scripts/change-log-generator.js

```diff
@@ -329,40 +373,43 @@ function writeChangeLog(textToInsert) {
   fs.writeSync(fd, buffer, 0, buffer.length, 0);
   fs.writeSync(fd, data, 0, data.length, buffer.length);
   fs.closeSync(fd);
+  console.log(`\nChange log written to: ${constants.CHANGE_LOG_PATH}`);
 }
 
 function openPRForChanges(releaseBranch, changeLogBranch) {
+  if (ADD_VERBOSE_LOGGING) {
+    console.log('\nOpening Pull Request for team review.');
+  }
   const commitCommand = `git commit -a -m "chore: generated CHANGELOG for ${releaseBranch}"`;
   const pushCommand = `git push origin ${changeLogBranch}`;
   shell.exec(commitCommand);
   shell.exec(pushCommand, { silent: true });
   shell.exec(
-    `open "https://github.com/forcedotcom/salesforcedx-vscode/pull/new/${changeLogBranch}"`
+    `open "https://github.com/forcedotcom/salesforcedx-vscode/pull/new/${releaseBranch}...${changeLogBranch}"`
```

Comment:
Automagically set the correct base branch. Very happy you mentioned this @brpowell 🥇